.ci/docker/common/install_clang.sh

```diff
@@ -21,9 +21,9 @@ if [ -n "$CLANG_VERSION" ]; then
 
   sudo apt-get update
   if [[ $CLANG_VERSION -ge 18 ]]; then
-    apt-get install -y libomp-${CLANG_VERSION}-dev libclang-rt-${CLANG_VERSION}-dev clang-"$CLANG_VERSION" llvm-"$CLANG_VERSION"
+    apt-get install -y libomp-${CLANG_VERSION}-dev libclang-rt-${CLANG_VERSION}-dev clang-"$CLANG_VERSION" llvm-"$CLANG_VERSION" lld-"$CLANG_VERSION"
   else
-    apt-get install -y --no-install-recommends clang-"$CLANG_VERSION" llvm-"$CLANG_VERSION"
+    apt-get install -y --no-install-recommends clang-"$CLANG_VERSION" llvm-"$CLANG_VERSION" lld-"$CLANG_VERSION" llvm-"$CLANG_VERSION"-tools
```

Comment:
How can In install llvmGold.so in versions less than llvm-11?

Reply:
Don't try on lower llvm

Reply:
I am not, but clang-10 doesn't seem to properly ship the binary llvmGold.so  or at the very least it's installed in the wrong place.